tools/bag_processing/scripts/convert_bayer_bag.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# Copyright (c) 2017, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+#
+# All rights reserved.
+#
+# The Astrobee platform is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""
+Converts bayer encoded images from the provided bagfile to grayscale images in a new bagfile.
+"""
+
+import argparse
+import os
+import sys
+
+import cv2
+import rosbag
+import rospy
+import utilities
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+
+
+# TODO(rsoussan): Update/remove comment when color conversion issue resolved.
```

Comment:
I'm noticing this comment is specifically about grayscale conversion, so it would make more sense if moved next to the line with `cv2.COLOR_BAYER_GR2GRAY`, and since we are more certain about the color channels, we could also clarify that `cv2.COLOR_BAYER_GB2GRAY` would be correct except for backward compatibility.